syntax = "proto3";

message ClientConfig {
    string value = 1;
}
